### ISSUE TITLE
Code example doesn`t work in Basic Concepts of Keras

### ISSUE THREAD

**Issue by Taranis71** [University-Clinic-of-Neuroradiology/python-bootcamp#39]
**Code example doesn`t work in Basic Concepts of Keras**

There is a code example in that exercise, that doesn't work.
If you execute the part "Changing Optimization Parameters" the event handler throws the following error:
_Argument `learning_rate` should be float, or an instance of LearningRateSchedule, or a callable (that takes in the current iteration value and returns the corresponding learning rate value). Received instead: learning_rate=1_ 

The problem is caused by this line of code:

![Image](https://github.com/user-attachments/assets/538a0ec7-1afb-429c-a173-733da47751d1)

To solve the problem change the argument **1** in the learningrates (integer) to **1.0** (float), like this:

![Image](https://github.com/user-attachments/assets/24cac114-8c60-4720-8437-781b930d268d)

After that correction the code examples should work.

I am using the following software components:
- Browser: Chrome
- Python IDE: Google Colab
